index.html

```diff
@@ -754,13 +754,14 @@ <h3>Validate Algorithm</h3>
       </ol>
 
       <p>
-When dereferencing a `statusListCredential` URL, implementations MAY provide a
-mechanism to query the status list at a particular point in time. If such a
+When a `statusListCredential` URL is dereferenced, server implementations MAY provide a
+mechanism to dereference the status list as of a particular point in time. If such a
 feature is supported, and if query parameters are supported by the URL scheme,
 then the name of the query parameter MUST be `timestamp` and the value MUST be
-a valid URL-encoded [[XMLSCHEMA11-2]] dateTimeStamp string value. The resulting
-status list credential SHOULD be the status list as it existed at the given
-date and time.
+a valid URL-encoded [[XMLSCHEMA11-2]] dateTimeStamp string value. The result of
+dereferencing such a timestamped URL MUST be a status list credential containing
+either the status list as it existed at the given point in time, or a 
+<a href="#STATUS_RETRIEVAL_ERROR">STATUS_RETRIEVAL_ERROR</a>.
```

Comment:
```suggestion
dereferencing such a timestamp-parameterized URL MUST be a status list credential
containing either the status list as it existed at the given point in time, or a 
<a href="#STATUS_RETRIEVAL_ERROR">STATUS_RETRIEVAL_ERROR</a>.
```

Reply:
or
```suggestion
dereferencing such a timestamp-parameterized URL MUST be either a status list
credential containing the status list as it existed at the given point in time,
or a <a href="#STATUS_RETRIEVAL_ERROR">STATUS_RETRIEVAL_ERROR</a>.
```

Reply:
Fixed in dfa5e3181daedba6c8ddefe8cc2026a0d1afe120.